app/controllers/find/v2/results_controller.rb

```diff
@@ -7,6 +7,7 @@ class ResultsController < Find::ApplicationController
 
       def index
         @courses = CoursesQuery.call(params:)
+        @courses_count = @courses.count
```

Comment:
Do we need this variable? It feels like we could either call `@courses.count` or `@pagy.count` in the view directly. Is it to cache the value?

Reply:
We use this count two times in the view and will send also for tracking. I think would be good to have this value.

For a while I was thinking that maybe courses query could have this value but we don't hold the courses query instance so I chosen this way.